build.gradle

```diff
@@ -17,6 +17,7 @@
 
 buildscript {
   repositories {
+    mavenLocal()
```

Comment:
Please don't commit this -- it is only for local test.

Reply:
Done.